FinniversKit/Sources/Components/Broadcast/Broadcast.swift

```diff
@@ -171,12 +172,20 @@ private extension Broadcast {
     }
 
     func insert(_ messages: Set<BroadcastMessage>) {
-        for message in messages {
+        var accessibilityLabelForContainer = ""
+        
+        for (index, message) in messages.enumerated() {
             let item = BroadcastItem(message: message)
+            let accessibleLabelForItem = "Alert \(index + 1), \(item.message.text)"
```

Comment:
Shouldn't `"Alert"` be localized?

Reply:
Yes if we want to keep it it will have to be provided from the caller 👍